src/main/java/com/gluonhq/substrate/Constants.java

```diff
@@ -138,6 +139,7 @@
     public static final String DALVIK_PRECOMPILED_CLASSES = "/precompiled/classes/";
     public static final String DALVIK_ACTIVITY_PACKAGE = "com/gluonhq/helloandroid/";
     public static final String DALVIK_JAVAFX_PACKAGE = "javafx/scene/input/";
```

Comment:
Unused constants, first one can be used in `prepareAndroidManifest`. Second one can be removed

Reply:
Manifest uses package name with dots as placeholder, so I removed this completely